src/main/scala/org/ua/ilm/datamining/crawlers/youtube/ParallelCrawlerChannels.scala

```diff
@@ -0,0 +1,21 @@
+package org.ua.ilm.datamining.crawlers.youtube
+
+import java.time.LocalTime
+import java.util.concurrent.{Callable, Executors}
+
+class ParallelCrawlerChannels(key: String, arrIdChannels: Array[String],
+                              partChannels: Option[String]) extends CrawlerChannels(key, arrIdChannels, partChannels) {
+
+  val service = Executors.newFixedThreadPool(2)
+
+
+  override def crawlerChannels(): Array[String] = {
```

Comment:
naming. crawler.crawlerChannels()? crawler is an entity, get/request is an action here.